styles.css

```diff
@@ -143,126 +189,211 @@ WORKS SECTION
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+
+.work-card__header img {
+  width: 100%;
+  height: 100%;
+  border-radius: 8px;
+  object-fit: contain;
+}
+
+.work-card__body {
+  width: 100%;
+}
+
+.work-card__body > ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: grid;
+  gap: 12px;
+}
+
+.work-card__title {
   padding: 0;
   margin: 0;
   font-weight: 700;
-  font-size: 32px;
-  line-height: 44px;
+  font-size: 2.2rem;
+  line-height: 2.2rem;
   color: #172b4d;
 }
 
-.work-card > .experience {
+.work-card__experience {
   list-style-type: none;
   padding: 6px 0;
   margin: 0;
   display: flex;
   gap: 12px;
 }
 
-.work-card > .experience > li {
+.work-card__experience > li {
   font-weight: 600;
-  font-size: 13px;
-  line-height: 16px;
+  font-size: 0.8rem;
+  line-height: 0.8rem;
   color: #7a869a;
 }
 
-.experience > li:first-child {
-  font-weight: 600;
-  font-size: 13px;
-  line-height: 16px;
-  text-transform: uppercase;
-  color: #344563;
-}
-
-.work-card > p {
+.work-card__description {
   margin: 0;
   padding: 0;
-  font-size: 15px;
-  line-height: 24px;
   color: #344563;
 }
 
-.work-card > .technologies {
+.work-card__technologies {
   list-style-type: none;
   padding: 3px 0;
   margin: 0;
   display: flex;
 }
 
-.work-card > .technologies > li {
+.work-card__technologies > li {
   background: #ebebff;
   border-radius: 8px;
   font-weight: 500;
-  font-size: 12px;
-  line-height: 16px;
+  font-size: 0.8rem;
+  line-height: 0.8rem;
   letter-spacing: 0.03em;
   color: #6070ff;
   padding: 4px 12px;
   margin-right: 8px;
 }
 
-#button-container {
+.work-card__experience > li:first-child {
+  font-weight: 600;
+  font-size: 0.8rem;
+  line-height: 0.8rem;
+  color: #344563;
+}
+
+.btn-container {
   width: 100%;
+  margin: 0;
   padding: 12px 0;
 }
 
-.btn {
+.btn-container > .btn {
   box-sizing: border-box;
   color: #396df2;
   border: 1px solid #6070ff;
   background: #fff;
   border-radius: 8px;
   width: max-content;
   font-weight: 500;
-  font-size: 17px;
-  line-height: 24px;
   letter-spacing: 0.03em;
   text-align: center;
   padding: 12px;
 }
 
-.btn > a {
+.btn-container > .btn > a {
   text-decoration: none;
   color: inherit;
 }
 
-.btn:hover {
+.btn-container .btn:hover {
   color: #fff;
   background: #6070ff;
 }
 
-.btn:active {
+.btn-container .btn:active {
   background: #2230d2;
   color: #fff;
   box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
   border: 1px solid #fff;
 }
 
-/*
-===========================
-ABOUT SECTION
-*/
+@media screen and (min-width: 768px) {
+  #works {
+    grid-template-columns: repeat(2, 1fr);
+    padding: 114px 10%;
+  }
+
+  .work-card {
+    grid-column: 1/3;
+  }
+
+  .work-card:hover {
+    filter: drop-shadow(0 48px 48px rgba(37, 47, 137, 0.08));
+    border: 1px solid #6070ff;
+  }
+
+  .work-card:hover .btn {
+    background: #4053fc;
+    color: #fff;
+    box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+    border: 1px solid #fff;
+  }
```

Comment:
- [ ] Here you are using a hover method on the card, instead of doing a hover on the card, you should do a hover on the button. Kindly change this hover functionality as you have on your mobile version, indeed, you shouldn't give a hover functionality to your desktop version. Since this is the same to mobile 💯

Reply:
Hi @BregornOriginal, thanks for your comments. I've looked at the design again in Figma, and I see the hover is on the card and not on the button. Please see the attached image from Figma - where the top card is with hover, and the bottom card is without hover - I see the mouse pointer is on the card and not on the button. Please have a look, and let me know if you still think I should set the hover on the button instead of the card. Thanks!

![Screen Shot 2022-12-08 at 2 19 39 PM](https://user-images.githubusercontent.com/17412822/206535669-7eaf7a64-bde9-4050-a7ec-ad03330f5ce4.png)